src/shader_recompiler/ir/passes/resource_tracking_pass.cpp

```diff
@@ -42,6 +42,17 @@ bool IsBufferInstruction(const IR::Inst& inst) {
     case IR::Opcode::StoreBufferFormatF32x3:
     case IR::Opcode::StoreBufferFormatF32x4:
     case IR::Opcode::StoreBufferU32:
+    case IR::Opcode::BufferAtomicIAdd32:
+    case IR::Opcode::BufferAtomicSMin32:
+    case IR::Opcode::BufferAtomicUMin32:
+    case IR::Opcode::BufferAtomicSMax32:
+    case IR::Opcode::BufferAtomicUMax32:
+    case IR::Opcode::BufferAtomicInc32:
+    case IR::Opcode::BufferAtomicDec32:
+    case IR::Opcode::BufferAtomicAnd32:
+    case IR::Opcode::BufferAtomicOr32:
+    case IR::Opcode::BufferAtomicXor32:
```

Comment:
These instructions need to be added to IsBufferStore() so the buffer handle is translated as storage buffer. Given they are used in many places I recommend you make a dedicated function IsBufferAtomic() and use it where needed